apps/condo/domains/user/oidc/adapter.js

```diff
@@ -0,0 +1,139 @@
+const Redis = require('ioredis')
+const { isEmpty } = require('lodash')
+
```

Comment:
Please, specify a url to a source code, this module was made on.
It seems, like it is based on some example.